charts/hail-search/templates/_helpers.tpl

```diff
@@ -1,25 +1,13 @@
-{{/*
-Expand the name of the chart.
-*/}}
-{{- define "hail-search.name" -}}
-{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
-{{- end }}
-
 {{/*
 Create a default fully qualified app name.
 We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
 If release name contains chart name it will be used as a full name.
 */}}
 {{- define "hail-search.fullname" -}}
-{{- if .Values.fullnameOverride }}
-{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
-{{- else }}
-{{- $name := default .Chart.Name .Values.nameOverride }}
-{{- if contains $name .Release.Name }}
+{{- if contains .Chart.Name .Release.Name }}
 {{- .Release.Name | trunc 63 | trimSuffix "-" }}
 {{- else }}
-{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
-{{- end }}
+{{- printf "%s-%s" .Release.Name .Chart.Name | trunc 63 | trimSuffix "-" }}
```

Comment:
release name is passed in [here](https://github.com/broadinstitute/seqr-private/blob/master/deployments/seqr/dev/seqr.yaml#L16)

Reply:
@sjahl actually.... for the sake of dogfooding this, how would we feel about changing the .Release.Name in our deployment to `broad`?  That'd be consistent with what we've put in our [readme](https://github.com/broadinstitute/seqr-helm/pull/54/files#diff-2d0cae1998d1122082f41d3a4aaace9eab74aa392851ddb1cb86450c1a79c408R58) and what we're likely to instruct external users to do when the process gets solidified.  

I know that doing that will for sure will require [this env var](https://github.com/broadinstitute/tgg-helm/blob/main/charts/lib-broad-seqr/values.yaml#L74) to change, and maybe more that are yet unknown?